.dependabot/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "monthly"
+
+  - package_manager: "docker"
+    directory: "/"
+    update_schedule: "monthly"
```

Comment:
We want dependabot to look at Dockerfiles. I'm pretty sure this will work. If not, hopefully we find out. This saves us from fiddling with minor Python version updates.